regexes/device/televisions.yml

```diff
@@ -699,3 +707,99 @@ Vestel:
   regex: '(?:Vestel.+VESTEL|VESTEL;)'
   device: 'tv'
   model: ''
+
+# Leff
+Leff:
+  # shell m-board
+  regex: 'Leff Shell'
+  device: 'tv'
+  model: ''
+
+# Leben
+Leben:
+  # shell m-board
+  regex: 'Leben Shell'
+  device: 'tv'
+  model: ''
+
+# Lumus
+Lumus:
+  # shell m-board
+  regex: 'LUMUS Shell'
+  device: 'tv'
+  model: ''
+
+# Erisson
+Erisson:
+  # shell m-board
+  regex: 'Erisson[_ ]Shell'
+  device: 'tv'
+  model: ''
+
+# BBK
+BBK:
+  # shell m-board
+  regex: 'BBK shell'
+  device: 'tv'
+  model: ''
+
+# Novex
+Novex:
+  # shell m-board
+  regex: 'Novex shell'
+  device: 'tv'
+  model: ''
+
+# Digma
+Digma:
+  regex: 'Digma Shell'
+  device: 'tv'
+  model: ''
+
+# AMCV
+AMCV:
+  regex: 'AMCV Shell'
+  device: 'tv'
+  model: ''
+
+# Mystery
+Mystery:
+  regex: 'Mystery Shell'
+  device: 'tv'
+  models:
+    - regex: 'Mystery Shell [0-9]+D([0-9]{2})([1-9])[A-Z]H[0-9]+_HD'
+      model: '$1.$2" HD Smart TV'
+    - regex: 'Mystery Shell [0-9]+D([0-9]{2})0[A-Z]F[0-9]+_FHD'
+      model: '$1" FHD Smart TV'
+
+# ECON (econ.su)
+ECON:
+  regex: 'ECON Shell'
+  device: 'tv'
+  model: ''
+
+# Starwind (starwind.com.ru)
+Starwind:
+  regex: 'Starwind Shell'
+  device: 'tv'
+  model: ''
+
+# Kvant (tvkvant.ru)
+Kvant:
+  regex: 'Kvant Shell'
+  device: 'tv'
+  model: ''
+
+# Hi
+Hi:
+  regex: 'Hi Shell'
+  device: 'tv'
+  model: ''
+
+# AKIRA (myakira.com)
+AKIRA:
+  regex: 'AKIRA Shell'
+  device: 'tv'
+  models:
+    - regex: 'AKIRA Shell [A-Z]{2}([0-9]{2})0[A-Z]{2}'
+      model: '$1" Smart TV'
```

Comment:
@sgiehl this is not an exact definition of the model name, from such a user agent we can only get the matrix size and screen type in some cases.
or is it better to leave the model empty string?

Reply:
Isn't something like `LC390TA2A` the model name in this case? Brands like LG actually have similar models.

Reply:
this is not the name of the device model, it is the name of the panel and the motherboard

https://www.panelook.com/LC390TA2A_Panda_39_CELL_overview_26807.html

this applies to the useragent in the following format
<brand> shell <panel code or framework>

* Mozilla/5.0 (Linux; Android 4.4.4; Mystery Shell 536D315LH71_HD) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.90 Safari/537.36
* Mozilla/5.0 (Linux; Android 4.4.4; JVC Shell 536D315LH71_HD Build/KTU84P) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/33.0.0.0 YaBrowser/19.5.0.155 (lite) Safari/537.36

* Mozilla/5.0 (Linux; Android 4.4.4; Mystery Shell 536D315XH33_HD) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.116 Safari/537.36
* Mozilla/5.0 (Linux; Android 4.4.4; JVC Shell 536D315XH33_HD Build/KTU84P) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/33.0.0.0 YaBrowser/19.5.0.155 (lite) Safari/537.36

* Mozilla/5.0 (Linux; Android 4.4.4; WildRed Shell V400HJ6_PE1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.116 Safari/537.36
* Mozilla/5.0 (Linux; Android 4.4.4; ERISSON_shell_V400HJ6_PE1_C3_40DN4_HD_W18023) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.143 Safari/537.36

Reply:
Ah ok. I see. Hm... I actually don't have a big preference in this case. Guess we can also leave it empty like for a couple of other TV brands, were the model can't be detected